src/callbacks/onPreprocess/updateRRatioSpan.js

```diff
@@ -0,0 +1,13 @@
+export default function updateRRatioSpan() {
```

Comment:
Again, I think we should avoid the axis distinction for R ratio and just stick with using labels.